sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/administration/DocumentModelAdministrationAsyncClient.java

```diff
@@ -6,19 +6,23 @@
 import com.azure.ai.formrecognizer.DocumentAnalysisAsyncClient;
 import com.azure.ai.formrecognizer.DocumentAnalysisClientBuilder;
 import com.azure.ai.formrecognizer.DocumentAnalysisServiceVersion;
-import com.azure.ai.formrecognizer.administration.models.ResourceInfo;
+import com.azure.ai.formrecognizer.administration.models.AzureBlobContentSourceT;
+import com.azure.ai.formrecognizer.administration.models.Base64ContentSourceT;
```

Comment:
Thinking we should expose base64 or local yet.

Reply:
@bojunehsu could you confirm the content source support when building models?